packages/gtk3.rb

```diff
@@ -59,16 +51,17 @@ def self.postinstall
     system "glib-compile-schemas #{CREW_PREFIX}/share/glib-2.0/schemas"
     # update mime database
     system "update-mime-database #{CREW_PREFIX}/share/mime"
-    puts
-    puts "To add basic settings, execute the following:".lightblue
-    puts "mkdir #{HOME}/.config/gtk-3.0".lightblue
-    puts "cat << 'EOF' > #{HOME}/.config/gtk-3.0/settings.ini
-[Settings]
-gtk-application-prefer-dark-theme = false
-gtk-icon-theme-name = hicolor
-gtk-fallback-icon-theme = gnome
-gtk-font-name = Arial 10
-EOF".lightblue
-    puts
+    
+    puts 'Performing configuration...'
+    FileUtils.mkdir_p("#{HOME}/.config/gtk-3.0")
+    @ini = "#{HOME}/.config/gtk-3.0/settings.ini"
+    @file = <<~EOF
+      [Settings]
+      gtk-application-prefer-dark-theme = false
+      gtk-icon-theme-name = hicolor
+      gtk-fallback-icon-theme = gnome
+      gtk-font-name = Arial 10
+    EOF
+    File.write(@ini, @file) unless File.exist?(@ini)
```

Comment:
Move to `def self.install` section?

Reply:
Actually, all you need is `File.write(@ini, @file)`.

Reply:
When all is said and done, we shouldn't need a `def self.postinstall` section.

Reply:
Maybe it makes sense to prompt to remove `#{HOME}/.config/gtk-3.0` in `def self.remove`?

Reply:
Moved to `install` section